### ISSUE TITLE
Error loading imdb_reviews on Julyter Lab and Windows 10 machine

### ISSUE THREAD

**Issue by wbadry** [tensorflow/datasets#2190]
```python
import numpy as np
import tensorflow as tf
import tensorflow_hub as hub
import tensorflow_datasets as tfds

print("Version: ", tf.__version__)
print("Eager mode: ", tf.executing_eagerly())
print("Hub version: ", hub.__version__)
print("GPU is", "available" if tf.config.experimental.list_physical_devices("GPU") else "NOT AVAILABLE")

```

```
Version:  2.2.0
Eager mode:  True
Hub version:  0.8.0
GPU is available
```

**The datasets** list can be loaded with no issues. When trying to **load imdb_reviews** or any other dataset, I encounter this error

```python
#tfds.list_builders()
imdb, info = tfds.load('imdb_reviews', with_info=True, as_supervised=True)
```

```python
---------------------------------------------------------------------------
UnimplementedError                        Traceback (most recent call last)
c:\python37\lib\site-packages\tensorflow_datasets\core\utils\py_utils.py in try_reraise(*args, **kwargs)
    398   try:
--> 399     yield
    400   except Exception:   # pylint: disable=broad-except

c:\python37\lib\site-packages\tensorflow_datasets\core\registered.py in builder(name, **builder_init_kwargs)
    243       prefix="Failed to construct dataset {}".format(name)):
--> 244     return builder_cls(name)(**builder_kwargs)
    245 

c:\python37\lib\site-packages\wrapt\wrappers.py in __call__(self, *args, **kwargs)
    602             return self._self_wrapper(self.__wrapped__, self._self_instance,
--> 603                     args, kwargs)
    604 

c:\python37\lib\site-packages\tensorflow_datasets\core\api_utils.py in disallow_positional_args_dec(fn, instance, args, kwargs)
     68     _check_required(fn, kwargs)
---> 69     return fn(*args, **kwargs)
     70 

c:\python37\lib\site-packages\tensorflow_datasets\core\dataset_builder.py in __init__(self, data_dir, config, version)
    205     else:  # Use the code version (do not restore data)
--> 206       self.info.initialize_from_bucket()
    207 

c:\python37\lib\site-packages\tensorflow_datasets\core\dataset_info.py in initialize_from_bucket(self)
    422     tmp_dir = tempfile.mkdtemp("tfds")
--> 423     data_files = gcs_utils.gcs_dataset_info_files(self.full_name)
    424     if not data_files:

c:\python37\lib\site-packages\tensorflow_datasets\core\utils\gcs_utils.py in gcs_dataset_info_files(dataset_dir)
     69   """Return paths to GCS files in the given dataset directory."""
---> 70   return gcs_listdir(posixpath.join(GCS_DATASET_INFO_DIR, dataset_dir))
     71 

c:\python37\lib\site-packages\tensorflow_datasets\core\utils\gcs_utils.py in gcs_listdir(dir_name)
     62   root_dir = gcs_path(dir_name)
---> 63   if _is_gcs_disabled or not tf.io.gfile.exists(root_dir):
     64     return None

c:\python37\lib\site-packages\tensorflow\python\lib\io\file_io.py in file_exists_v2(path)
    266   try:
--> 267     _pywrap_file_io.FileExists(compat.as_bytes(path))
    268   except errors.NotFoundError:

UnimplementedError: File system scheme 'gs' not implemented (file: 'gs://tfds-data/dataset_info/imdb_reviews/plain_text/1.0.0')

During handling of the above exception, another exception occurred:

TypeError                                 Traceback (most recent call last)
<ipython-input-33-06930b64f980> in <module>
      1 #tfds.list_builders()
----> 2 imdb, info = tfds.load('imdb_reviews', with_info=True, as_supervised=True)

c:\python37\lib\site-packages\wrapt\wrappers.py in __call__(self, *args, **kwargs)
    562 
    563         return self._self_wrapper(self.__wrapped__, self._self_instance,
--> 564                 args, kwargs)
    565 
    566 class BoundFunctionWrapper(_FunctionWrapperBase):

c:\python37\lib\site-packages\tensorflow_datasets\core\api_utils.py in disallow_positional_args_dec(fn, instance, args, kwargs)
     67     _check_no_positional(fn, args, ismethod, allowed=allowed)
     68     _check_required(fn, kwargs)
---> 69     return fn(*args, **kwargs)
     70 
     71   return disallow_positional_args_dec(wrapped)  # pylint: disable=no-value-for-parameter

c:\python37\lib\site-packages\tensorflow_datasets\core\registered.py in load(name, split, data_dir, batch_size, shuffle_files, download, as_supervised, decoders, read_config, with_info, builder_kwargs, download_and_prepare_kwargs, as_dataset_kwargs, try_gcs)
    366     data_dir = constants.DATA_DIR
    367 
--> 368   dbuilder = builder(name, data_dir=data_dir, **builder_kwargs)
    369   if download:
    370     download_and_prepare_kwargs = download_and_prepare_kwargs or {}

c:\python37\lib\site-packages\tensorflow_datasets\core\registered.py in builder(name, **builder_init_kwargs)
    242   with py_utils.try_reraise(
    243       prefix="Failed to construct dataset {}".format(name)):
--> 244     return builder_cls(name)(**builder_kwargs)
    245 
    246 

c:\python37\lib\contextlib.py in __exit__(self, type, value, traceback)
    128                 value = type()
    129             try:
--> 130                 self.gen.throw(type, value, traceback)
    131             except StopIteration as exc:
    132                 # Suppress StopIteration *unless* it's the same exception that

c:\python37\lib\site-packages\tensorflow_datasets\core\utils\py_utils.py in try_reraise(*args, **kwargs)
    399     yield
    400   except Exception:   # pylint: disable=broad-except
--> 401     reraise(*args, **kwargs)
    402 
    403 

c:\python37\lib\site-packages\tensorflow_datasets\core\utils\py_utils.py in reraise(prefix, suffix)
    390   suffix = '\n' + suffix if suffix else ''
    391   msg = prefix + str(exc_value) + suffix
--> 392   six.reraise(exc_type, exc_type(msg), exc_traceback)
    393 
    394 

TypeError: __init__() missing 2 required positional arguments: 'op' and 'message'
```

May I know the problem? Thanks

**Comment by vijayphoenix** [comment]
Similar issue https://github.com/tensorflow/datasets/issues/2191

**Comment by Conchylicultor** [comment]
As first step, we should disable GCS loading on Windows. @vijayphoenix @sharanramjee could you have a look to fix this ?

**Comment by vijayphoenix** [comment]
Yeah, it works when we set this flag to True forcefully

https://github.com/tensorflow/datasets/blob/b37f6dc8ad85b81f0b2de2dde1c8ceca070942db/tensorflow_datasets/core/utils/gcs_utils.py#L40

**Comment by Conchylicultor** [comment]
Sorry for the breakage, it was introduced by using tf.io.gfile instead of request when we released `3.2.0`. In the meantime, the issue could be solved by downgrading to `3.1.0`.

**Comment by wbadry** [comment]
> Sorry for the breakage, it was introduced by using tf.io.gfile instead of request when we released `3.2.0`. In the meantime, the issue could be solved by downgrading to `3.1.0`.

Thanks @Conchylicultor . This is exactly what I tried yesterday and it worked.

**Comment by rsepassi** [comment]
v3.2.1 now released with the fix
